src/components/SearchInput.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import { FilmList } from '../fixtures/FilmList.js'
+
+export default class SearchInput extends Component {
+  render() {
+    return (
+      <div>
+          <div>Search:</div>
+          <input placeholder='type here'/>
+          { FilmList.map((val, key) => (
+              <div>
+                <p key="{film}">{val.name}</p>
```

Comment:
You should use id for key value

Reply:
thank you, fixed in https://github.com/AlexeyAndrianov/cinemavision/pull/1/commits/72f66d1c93669bff738ba0df8dd38fbb962a8fc2